server.js

```diff
@@ -206,6 +206,7 @@ app.use('/pl/shibcallback', require('./pages/authCallbackShib/authCallbackShib')
 app.use('/pl/azure_login', require('./pages/authLoginAzure/authLoginAzure'));
 app.use('/pl/azure_callback', require('./pages/authCallbackAzure/authCallbackAzure'));
 app.use('/pl/lti', require('./pages/authCallbackLti/authCallbackLti'));
+app.use('/pl/devlogin', require('./pages/authLoginDev/authLoginDev'));
```

Comment:
Should we wrap this in `if (config.devMode) {}`?

Reply:
We could, but the first line of that file checks for devMode and errors.

Reply:
https://github.com/PrairieLearn/PrairieLearn/blob/e214adadb9f3fdbdb93b30819aa99b36deb19a3a/pages/authLoginDev/authLoginDev.js#L10

Reply:
Yep, I checked 😄. But I find it nice when skimming through `server.js` to see what is mounted in different modes: https://github.com/PrairieLearn/PrairieLearn/blob/517be1007699db7ab2d5c823a5912fa00b39d045/server.js#L241

I would also keep the check in place in the page code too. We _really_ don't want this route to accidentally be enabled in prod!

Reply:
It's consistent with how the other auth methods work if they're enabled or not (we don't alter the route, just if it fires anything in the target page). I'm okay to add a check in server.js, though, if the skimming makes it more obvious?

Reply:
It's not crucial, but I do like to be able to easily see what's active when we are in prod versus dev.